website2/docs/getting-started-local-single-node.md

```diff
@@ -20,7 +20,7 @@ sidebar_label: Local (Single Node)
     under the License.
 -->
 
-> The current version of Heron is **{{% heronVersion %}}**
+> The current version of Heron is {{heron:version}}
```

Comment:
when I open the webpage locally, the line is "The current version of Heron is {{heron:version}}". The heron:version is not resolved. Same thing happens to pyversion I think.

Is this expected?

Reply:
Ya, we discussed offline to fix the versions like heron versions and bazel versions in docs after this PR is done...

Reply:
Sounds good to me~ Let me approve and merge it. Thanks.